configs/centerpoint/centerpoint_0075voxel_second_secfpn_4x8_cyclic_20e_nus.py

```diff
@@ -0,0 +1,206 @@
+_base_ = [
+    '../_base_/models/centerpoint_01voxel_second_secfpn_nus.py',
+    '../_base_/default_runtime.py'
```

Comment:
We can inherit `nus-3d.py` as `base` and skip some duplicated keys such as path and data_root.